internal/runners/hello/hello_example.go

```diff
@@ -58,29 +56,48 @@ func New(p primeable) *Hello {
 	}
 }
 
+// processError contains the scope in which errors are processed. This is
+// useful for ensuring that errors are wrapped in a user-facing error and
+// localized.
+func processError(err *error) {
+	if err != nil {
```

Comment:
Just return early `if err == nil`, so this whole block doesn't need to be indented.